features/admin.applications.v1/pages/applications.tsx

```diff
@@ -260,6 +296,40 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
         }));
     }, [ myAccountApplicationDataFetchRequestError ]);
 
+    /**
+     * Handles the My Account application data fetch request error.
+     */
+    useEffect(() => {
+        if (!myAccountApplicationGetRequestError) {
+            return;
+        }
+
+        if (
+            myAccountApplicationGetRequestError?.response?.data?.description
+        ) {
+            if (myAccountApplicationGetRequestError.response?.status === 404) {
+                return;
+            }
```

Comment:
We should ignore 404 only in legacy runtime right?

Reply:
This is to handle any 404 error case in general.